cinex/Services/AppService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using cinex.Models;
+using RestSharp;
+using System.Threading.Tasks;
+
+namespace cinex.Services
+{
+    public class AppService : IAppService
+    {
+        private static string LANGUAGE = "en-US";
+        private static string API_KEY = "1f54bd990f1cdfb230adb312546d765d";
+
+        public AppService()
+        {
+
+        }
+
+        public async Task<GenresResponseModel> GetGenresAsync()
+        {
+            var client = new RestClient($"https://api.themoviedb.org/3/genre/movie/list?language={LANGUAGE}&api_key={API_KEY}");
```

Comment:
Those URL values could be in a constant file.

Reply:
Yes, I moved to the AppConstants class.